package.json

```diff
@@ -14,7 +14,8 @@
         "apify": "^1.2.1",
         "moment": "^2.29.1",
         "node-fetch": "^2.6.1",
-        "puppeteer": "*"
+        "puppeteer": "*",
+        "apify-utils": "https://github.com/metalwarrior665/apify-utils.git#1.0.1"
```

Comment:
use `1.1.0`

Reply:
We decided to copy the functions before we finalize the package.